sites/feedback/includes/feedback.html

```diff
@@ -0,0 +1,9 @@
+{%- if page.feedbackPath -%}
+	<div data-ajax-replace= "{{page.feedbackPath}}">
+		<a class="btn btn-default btn-block" href="{{ i18nText-feedbackLink }}">{{ i18nText-feedback }}</a>
+	</div>
+{%- else -%}
+	<div>
+		<a class="btn btn-default btn-block" href="{{ i18nText-feedbackLink }}">{{ i18nText-feedback }}</a>
+	</div>
+{%- endif -%}
```

Comment:
Optional - may be an alternate version like this? It avoid to code twice the anchor.
```
<div {% if page.feedbackPath -%}
	data-ajax-replace= "{{page.feedbackPath}}"
{%- endif -%}
>
	<a class="btn btn-default btn-block" href="{{ i18nText-feedbackLink }}">{{ i18nText-feedback }}</a>
</div>
```

Reply:
Changed. Thanks